src/services/invoices/invoicesService.test.ts

```diff
@@ -0,0 +1,96 @@
+import { test, expect, describe, beforeAll, afterEach, afterAll } from 'vitest';
+import InvoicesService from './invoicesService.ts';
+import { server } from '../../mocks/server.ts';
+import { mockInvoice } from '../../mocks/invoice.mock.ts';
+
+const MOCK_INVOICE = mockInvoice;
+
+describe('invoicesService', () => {
+  beforeAll(() => server.listen());
+  afterEach(() => server.resetHandlers());
+  afterAll(() => server.close());
+
+  describe('fetchAllInvoices', () => {
+    test('returns invoices data when invoice exists', async () => {
+      const data = await InvoicesService.fetchAllInvoices();
```

Comment:
These tests mostly test if msw is working correctly :D I'd rather treat it as an implementation detail, user doesn't care if you are calling GET / POST or PUT, he/she just wants to get and modify data :D. So I'd rather suggest testing the whole InvoiceList / AddInvoice / EditInvoice page which calls MSW and gets data instead.